src/main/kotlin/de/bund/digitalservice/useid/identification/IdentificationSessionService.kt

```diff
@@ -41,7 +41,7 @@ class IdentificationSessionService(
     }
 
     private fun buildTcTokenUrl(session: IdentificationSession): String {
-        return "${applicationProperties.baseUrl}$IDENTIFICATION_SESSIONS_BASE_PATH/${session.useIdSessionId}/$TCTOKEN_PATH_SUFFIX"
+        return "${applicationProperties.baseUrl}$IDENTIFICATIONS_OLD_BASE_PATH/${session.useIdSessionId}/$TCTOKEN_PATH_SUFFIX"
```

Comment:
This should be `IDENTIFICATION_SESSIONS_BASE_PATH` (the new endpoint) right?

Reply:
Yes, after the whole migration. However, this would only change with #248. Therefore, I left the old base path here as the tc token endpoints are not changed here